pkg/metrics/prometheus_test.go

```diff
@@ -30,11 +30,12 @@ func apisixBadStatusCodesTestHandler(t *testing.T, metrics []*io_prometheus_clie
 		assert.Equal(t, metric.Type.String(), "GAUGE")
 		m := metric.GetMetric()
 		assert.Len(t, m, 2)
+
 		assert.Equal(t, *m[0].Gauge.Value, float64(1))
 		assert.Equal(t, *m[0].Label[0].Name, "controller_namespace")
 		assert.Equal(t, *m[0].Label[0].Value, "default")
 		assert.Equal(t, *m[0].Label[1].Name, "controller_pod")
-		assert.Equal(t, *m[0].Label[1].Value, "test")
```

Comment:
since we put  `podName` and `podNamespace` inside `NewPrometheusCollector()`, transfer param `"test"` to collector is disabled. 
so the test result of `controller_pod` should be `""`.